README.example.md

```diff
@@ -51,32 +53,83 @@ This section should include any business related explanation that helps understa
 If the project is using some special external services (NemID, oAuth, Customer API, etc.) give a short description here.
 Example: The app is using NemID to authenticate all its users.
 
-# Requirements
+# Getting the project running
 
-You must have the following installed and available on your machine:
+There are three supported ways to run the project:
 
-- **Ruby 3.0.x**
-- **Node JS 16.x**
-- **Yarn 1.x**
-- **PostgreSQL 12**
-- **Redis**
+1. [Run locally using docker for database servers](#run-locally-using-docker-for-database-servers)
+2. [Run entirely project locally (for non-docker purists)](#run-entirely-project-locally)
+3. [Run inside docker dev container (for docker purists)](#run-inside-docker-dev-container)
 
-## Installing requirements
+## Run locally using docker for database servers
 
-With [asdf](https://asdf-vm.com/) and [docker](https://www.docker.com/) (or [podman](https://podman.io/)) you can easily install/run the required versions of runtimes and services.
+Requirements:
 
-Install runtimes with `asdf`:
+- docker (or podman)
+- asdf
 
-```
+1. First install node and ruby:
+
+```bash
+# Add nodejs and ruby plugins if you don't already have them
+asdf plugin add nodejs
+asdf plugin ad ruby
+
+# Install
 asdf install
```

Comment:
Normally after I install a new nodejs version and the project is using yarn I have to run `npm -g install yarn` but it might be just my setup.

Also when I just run `asdf install` it will try to install some old stuff from other projects like 
python, java

```dotnet-core 6.0.400 is already installed
java zulu-jre-javafx-19.30.11 is already installed
Trying to update node-build... ok
Downloading node-v18.17.1-darwin-arm64.tar.gz...
-> https://nodejs.org/dist/v18.17.1/node-v18.17.1-darwin-arm64.tar.gz

WARNING: node-v18.17.1-darwin-arm64 is in LTS Maintenance mode and nearing its end of life.
It only receives *critical* security updates, *critical* bug fixes and documentation updates.

Installing node-v18.17.1-darwin-arm64...
Installed node-v18.17.1-darwin-arm64 to /Users/reinis/.asdf/installs/nodejs/18.17.1

python 3.10.5 is already installed
python 2.7.18 is already installed
ruby 3.2.2 is already installed
```

I assume you dont get such messages.
I normally install tool directly by running `asdf install nodejs 18.17.1`

Reply:
> Normally after I install a new nodejs version and the project is using yarn I have to run npm -g install yarn but it might be just my setup.

Great point, I'll add that :+1: 

> Also when I just run asdf install it will try to install some old stuff from other projects like
python, java

I think it lists other tool chains because you have them in a .tool-versions somewhere up the folder hierarchy, could be `~/.tool-versions`.